lib/jira/commands/checkout.rb

```diff
@@ -0,0 +1,88 @@
+module Jira
+  class CLI < Thor
+
+    desc "checkout", "Checks out a ticket from JIRA in the git branch"
+    method_option :remote, aliases: "-r", type: :string, default: nil, lazy_default: "", banner: "REMOTE"
+    def checkout(ticket=Jira::Core.ticket)
```

Comment:
can we remove the default? dont think it is applicable in this context - since we're trying to check out a new issue, rather than the defaulting to the existing issue